README.md

```diff
@@ -41,6 +41,15 @@ Generic scripts that don't need env vars. Use these for development:
 | npm run unittest:dev   |  Runs the unittests and watches for changes         |
 | npm run servertest     |  Runs the servertests                               |
 
+### Code coverage
+
+The `npm run unittest` command generates a report of how well the tests
```

Comment:
Might be worth being explicit here that this is for unittests not the server tests?